aws/tkg_vpc/jumpbox.tf

```diff
@@ -93,7 +93,11 @@ resource "aws_instance" "ubuntu" {
     ]
   }
   provisioner "file" {
-    source      = "./tkg_vpc/script.sh"
+    content     = templatefile("./tkg_vpc/templates/script.tpl",
```

Comment:
looking inside script.tpl, it totally a bash script, so I would like it if we can name it to .sh. For other developers' sake. Because tpl is also not terraform recommend extension name.

See:
*.tftpl is the recommended naming pattern to use for your template files. Terraform will not prevent you from using other names

Reply:
I was following the existing `.tpl` pattern being used within the templates folder, but i agree that `.sh` makes more sense and is less confusing.

thanks for the suggestion.